media/css/protocol/components/newsletter-form.scss

```diff
@@ -42,4 +42,19 @@ $image-path: '/media/protocol/img';
     }
 }
 
-/* stylelint-enable declaration-no-important */
+// Honeypot field styling - hide from users but keep accessible to bots
+.super-priority-field {
+    position: absolute !important;
+    left: -9999px !important;
+    top: -9999px !important;
+    width: 1px !important;
+    height: 1px !important;
+    overflow: hidden !important;
+    opacity: 0 !important;
+    pointer-events: none !important;
+
+    // Hide from screen readers
+    label {
+        display: none !important;
+    }
+}
```

Comment:
Nit: there might be existing mixins with the same outcome, or at least to save some of the boilerplate. Look for "`visually-hidden`" and similar in Protocol.